### PR TITLE
Feat/integrate torus

### PR DESCRIPTION
**Pull request by YZhenY**

### CODE REVIEW

File: app/config.js [hunk 1]
```diff
@@ -79,6 +79,15 @@ export default {
       allowed: true,
       buttonText: 'Connect',
       securityLevel: 3
+    },
+    {
+      id: 1007,
+      img: 'torus.svg',
+      title: 'Google',
+      body: 'Access your via Google via Torus',
```
Comment (evesnow91) [inline app/config.js:87]: Access your wallet via Google using Torus

**Comment by man15h** [comment]
Will review it and merge after ZRC2 and XSGD integration.

**Comment by YZhenY** [comment]
before merging, we need to shift to torus mainnet. for that we would require a google client_id from you guys

**Comment by YZhenY** [comment]
bumping this @evesnow91

**Pull request merged**